zulipterminal/model.py

```diff
@@ -278,6 +278,32 @@ def get_focus_in_current_narrow(self) -> Union[int, Set[None]]:
     def set_focus_in_current_narrow(self, focus_message: int) -> None:
         self.index["pointer"][repr(self.narrow)] = focus_message
 
+    def addtional_search_text_for_current_narrow(self) -> str:
+        """
+        This function is to check if the user is currently inside a narrow or not
+        and if they are, then the additional_search text is created accordingly
+        """
+        # In cases where '+' is already present in stream and/or topic name, it is converted to '%2B'
+        # then the spaces in stream and topic names are converted to '+'
+        additional_search_query_text = ""
```

Comment:
Minor: The logic here is fairly simple and sets just one primary name, so a much shorter variable name should be fine, within reason - and make it easier to read :) The text gets returned as the name of the function, so the function name can explain what the 'additional_search_query_text` means on it's own.